jsyntrax/src/main/java/org/atpfivt/jsyntrax/styles/StyleConfig.java

```diff
@@ -0,0 +1,151 @@
+package org.atpfivt.jsyntrax.styles;
+
+import org.ini4j.Profile.Section;
+import org.ini4j.Wini;
+
+import java.awt.*;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+import java.util.regex.Pattern;
+
+public class StyleConfig {
+    public StyleConfig() {
+        nodeStyles = new ArrayList<>(List.of(
+                new NodeBubbleStyle(),
+                new NodeBoxStyle(),
+                new NodeTokenStyle(),
+                new NodeHexStyle()));
+    }
+
+    public StyleConfig(Path style) {
+        this();
+        try {
+            Wini ini = new Wini(style.toFile());
+            if (ini.containsKey("style")) {
+                parseStyleArgs(ini.get("style"));
+            }
+
+            nodeStyles.clear();
+
+            for (Section s : ini.values()) {
+                if (s.getName().equals("style")) {
+                    continue;
+                }
+                // this is custom node style
+                NodeStyle ns = new NodeStyle();
+                ns.name = s.getName();
+                parseNodeStyle(s, ns);
+                nodeStyles.add(ns);
+            }
+        }
+        catch(NoSuchFieldException | IllegalAccessException | IOException e) {}
+    }
+
+    private void parseStyleArgs(Section s) throws NoSuchFieldException, IllegalAccessException {
+        parseField(s, this, "line_width", Integer::parseInt);
+        parseField(s, this, "outline_width", Integer::parseInt);
+        parseField(s, this, "padding", Integer::parseInt);
+        parseField(s, this, "line_color", StyleConfig::colorFromString);
+        parseField(s, this, "max_radius", Integer::parseInt);
+        parseField(s, this, "h_sep", Integer::parseInt);
+        parseField(s, this, "v_sep", Integer::parseInt);
+        parseField(s, this, "arrows", Boolean::parseBoolean);
+        parseField(s, this, "title_pos", (String v) -> v.substring(1, v.length() - 1));
```

Comment:
Why? I think title_pos must be an enum with values `tl`, `tc`, `tr`, `bl`, `bc`, `br`, no?